res/values-es/strings.xml

```diff
@@ -5,27 +5,27 @@
     <string name="Apply">Aplicar</string>
     <string name="BangButtonExplanation">Buscar en otros lugares es fácil con !comandos</string>
     <string name="Cancel">Cancelar</string>
-    <string name="ChangingFontSize">Cambiando Tamaño de Letra&#8230;</string>
+    <string name="ChangingFontSize">Cambiando tamaño de letra&#8230;</string>
     <string name="Confirm">Confirmar</string>
-    <string name="ConfirmClearCacheAndCookies">El caché del navegador va a ser eliminado. ¿Estás seguro?</string>
-    <string name="ConfirmClearCookies">Todas las cookies del navegador serán eliminadas. ¿Estás seguro?</string>
-    <string name="ConfirmClearHistory"> Todo el historial de búsqueda será eliminado. ¿Estás seguro? </string>
-    <string name="ConfirmExternalBrowser">Por favor Confirma</string>
-    <string name="Continue">Continua</string>
+    <string name="ConfirmClearCacheAndCookies">Se eliminará el caché del navegador. ¿Estás seguro?</string>
+    <string name="ConfirmClearCookies">Se eliminarán todas las cookies del navegador. ¿Estás seguro?</string>
+    <string name="ConfirmClearHistory">Se eliminará todo el historial de búsqueda. ¿Estás seguro?</string>
+    <string name="ConfirmExternalBrowser">Por favor, confirma</string>
+    <string name="Continue">Continuar</string>
     <string name="Default">Predeterminado</string>
-    <string name="Defaults"> Restaurar ha Predeterminado </string>
-    <string name="DefaultsSetToast">Fuesntes restauradas a predeterminado</string>
+    <string name="Defaults">Restaurar ha predeterminado</string>
+    <string name="DefaultsSetToast">Se restauraron las fuentes predeterminadas</string>
     <string name="ErrorActivityNotFound">No se encontró actividad relacionada</string>
-    <string name="ErrorAppNotFound">Applicaón no encontrada</string>
-    <string name="ErrorDownloadFailed">¡Descarga fallo!</string>
+    <string name="ErrorAppNotFound">Aplicación no encontrada</string>
+    <string name="ErrorDownloadFailed">¡Descarga fallida!</string>
     <string name="ErrorDownloadOpenFail">¡No fue posible abrir los archivos descargados!</string>
-    <string name="ErrorFeedDetail">Petición fallida, posiblemente un problema de conexión. Podrás navegar artículos pre cargados o puedes escoger reintentar.</string>
+    <string name="ErrorFeedDetail">Petición fallida, posiblemente por un problema de conexión. Puedes navegar en artículos pre-cargados o puedes reintentar.</string>
     <string name="ErrorFeedTitle">Error: Petición</string>
     <string name="FeedbackSubject">DuckDuckGo Android v2</string>
-    <string name="FeedbackTo">DuckDuckGo Retroalimentación &lt;help@duckduckgo.com&gt;</string>
-    <string name="GetStarted">Comienza</string>
+    <string name="FeedbackTo">Retroalimentación de DuckDuckGo &lt;help@duckduckgo.com&gt;</string>
```

Comment:
"Retroalimentación" is not used in Spanish. The English word is Ok in this case: "Feedback para DuckDuckGo" or maybe "Comentarios a DuckDuckGo" if you want to go all Spanish.